app/src/main/resources/config/leagur-app.yml

```diff
@@ -0,0 +1,25 @@
+##### Application Configuration #####
+# This file contains application level configuration.
+#
+# Application configuration is consumed natively by
+# the DropWizard application framework.
+#
+# For a reference as to what can be specified here, see this:
+# https://dropwizard.github.io/dropwizard/manual/configuration.html
```

Comment:
I'd advise using this framework called DropWizard. This is a basic application level configuration file for it. The link above talks about more configuration that you can provide.